x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/alerts.ts

```diff
@@ -502,19 +501,6 @@ instanceStateValue: true
               })
             );
 
-          // Enqueue non ephemerically so we the latter code can query properly
```

Comment:
Removing this is the fix for the `superuser at space1 should handle custom retry logic when appropriate` flaky test. Discussed with @chrisronline and this was something that was added to update the tests when ephemeral actions were enabled. Since the tests are running with ephemeral disabled, this should have been removed but was overlooked. After removing, I no longer see flakiness in the flaky test runner.